q2_composition/_ancom.py

```diff
@@ -98,25 +98,47 @@ def ancom(output_dir: str,
 
     significant_features = ancom_results[0][
         ancom_results[0]['Reject null hypothesis']]
+    significant_features_present = not significant_features.empty
+    insignificant_table = ('<table><tr><th></th><th>W</th></tr><tr>'
+                           '<td>No significant features identified!</td>'
+                           '<td>NA</td></tr></table>')
 
     with open(index_fp, 'w') as index_f:
-        index_f.write('<html><body>\n')
+        index_f.write('<html>\n')
+        if html is not None:
+            index_f.write('<head>\n')
+            index_f.write(INLINE.render())
+            index_f.write('</head>\n')
+        index_f.write('<body>\n')
         index_f.write('<h1>ANCOM statistical results</h1>\n')
-        index_f.write('<a href="ancom.csv">Download as CSV</a><br>\n')
-        index_f.write(q2templates.df_to_html(
-            significant_features['W'].to_frame(), border=None, classes=None))
+        index_f.write('<a href="ancom.csv">Download complete table as CSV</a>'
+                      '<br>\n')
+        if significant_features_present:
+            index_f.write(q2templates.df_to_html(
+                significant_features['W'].to_frame(), border=None,
+                classes=None))
+        else:
+            index_f.write(insignificant_table)
         if len(ancom_results) == 2:
             ancom_results[1].to_csv(os.path.join(output_dir,
                                                  'percent-abundances.csv'),
                                     header=True, index=True)
             index_f.write(('<h1>Percentile abundances of features '
                            'by group</h1>\n'))
             index_f.write(('<a href="percent-abundances.csv">'
-                           'Download as CSV</a><br>\n'))
-            index_f.write(q2templates.df_to_html(
-                ancom_results[1].loc[significant_features.index],
-                border=None, classes=None))
-        index_f.write(html)
+                           'Download complete table as CSV</a><br>\n'))
+            if significant_features_present:
+                index_f.write(q2templates.df_to_html(
+                    ancom_results[1].loc[significant_features.index],
+                    border=None, classes=None))
+            else:
+                index_f.write(insignificant_table)
```

Comment:
This table has a different structure than the percentile abundances which looks like of strange. Maybe instead of making this a table, it is just a note, that way we can use the same note in both places.